RELEASE_PROCESS.md

```diff
@@ -72,18 +152,34 @@ The git SHA of the build you are testing can be found in
 ### Building and Releasing the Release
 
 - [ ] Select a version from the `current` channel that you wish to promote to `stable`. Make sure that this version has gone through the upgrade testing.
-- [ ] Use expeditor to promote the build:
+- [ ] Use expeditor to promote the build.  The expeditor command is of the form:
+
+        /expeditor promote ORG/REPO:BRANCH VERSION
+
+In practice it will look like:
 
         /expeditor promote chef/chef-server:master VERSION
 
-  Please do this in the `#chef-server` room. Once this is
+Example:
+```
+/expeditor promote chef/chef-server:master 14.2.2
+```
+
+  Please do this in the `#chef-server-notify` channel.  Once this is
   done, the release is available to the public via the APT and YUM
   repositories and downloads.chef.io.
 
 - [ ] Chef employees should already know a release is coming; however, as a
   courtesy, drop a message in the #cft-announce slack channel that the release
   is coming. Provide the release number and any highlights of the release.
 
+### Announce the release post-promote to the following channels:
+    - #a2-release-coordinate
+    - #chef-server
+    - #cft-announce  
+Copying/pasting a discourse link to the post-promote channels should suffice.  You can find the link here.  Note that this is NOT the link to copy/paste, this is a link where you can find the link to copy/paste:  
+https://discourse.chef.io/c/chef-release/9
+
 - [ ] Write and then publish a Discourse post on https://discourse.chef.io
```

Comment:
We no longer need to perform this step. Expeditor published the release notes to disourse for us.